app/Http/Controllers/Admin/HoldAdminController.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Models\Hold\Hold;
+use App\Models\Navigation\Navaid;
+use Illuminate\Http\JsonResponse;
+use App\Http\Requests\HoldRequest;
+use App\Services\DependencyService;
+use App\Http\Controllers\BaseController;
+
+class HoldAdminController extends BaseController
+{
+    private const DESCRIPTION_NOT_UNIQUE_RESPONSE = ['message' => 'Description of hold already used.'];
+    private const HOLD_NOT_IN_NAVAID_RESPONSE = ['message' => 'Hold not associated with Navaid.'];
+    /**
+     * Get a list of holds for a specified Navaid.
+     *
+     * @param Navaid $navaid
+     * @return JsonResponse
+     */
+    public function getHolds(Navaid $navaid) : JsonResponse
+    {
+        $holds = $navaid->load(['holds'])->holds;
```

Comment:
Is is possible to use dot notation here to load them all at once, e.g. `$holds = $navaid->load(['holds', 'holds.restrictions'])->holds;`

Reply:
https://github.com/VATSIM-UK/uk-controller-api/pull/520/commits/2c888cb0c3cc358be8ffedd163376aea2cbcb004

(nice call!)